app/views/stats/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1>Statistics</h1>
+<h1>Statistics<%=" for #{@organization.name}" unless @organization.nil?%> </h1>
```

Comment:
Old functionality to show all courses (for all organizations) stats in /stats was still retained. This was also kept without permission checks, based on irc discussions.

Do we want to get rid of that? Even for admin purposes? Can change if needed.